gh-680 Restyle footer to always show MDM logo

pjmonks opened the pull request:
- Adjust footer links alignment
- Add copyright and "Powered by MDM" text/logo to right side

Resolves #681 

![image](https://user-images.githubusercontent.com/3219480/204318615-b56c3c91-b1a8-468b-97c0-28bac494b854.png)

jamesrwelch commented:
This looks good on a large screen, but a bit odd on a smaller one... maybe the right margin needs fixing?  And do the LHS links want to be horizontal or vertical? 
<img width="668" alt="Screenshot 2022-11-30 at 12 43 43" src="https://user-images.githubusercontent.com/1869668/204799790-8d5d3615-ebb0-4087-853d-d03dd022e4c6.png">

pjmonks commented:
@jamesrwelch Updates to this PR made:

- Rebased from latest `develop` changes using Angular 14
- Incorporated your feedback:

Desktop view:

![image](https://user-images.githubusercontent.com/3219480/215110505-66f1ff90-bc17-4bc8-ac12-66ff3caf481b.png)

Table/Mobile view:

![image](https://user-images.githubusercontent.com/3219480/215110638-157f1a5b-696b-46e5-b3bb-9c2027bb9ac0.png)

jamesrwelch reviewed (approved):
This works really nicely - thank you for the recent changes
